src/TestingFramework/TestCase/UnitTestCase.php

```diff
@@ -82,6 +66,21 @@ protected function tearDown()
                 throw new \RuntimeException('tearDown() cleanup: File, link or directory does not exist', 1410633510);
             }
         }
-        $this->testFilesToDelete = [];
+
+        // Unset properties of test classes to safe memory
```

Comment:
Spelling: This should be "save" (the verb), not "safe".

Reply:
Taken from Core classes ;-)